public/scripts/extensions/vectors/index.js

```diff
@@ -745,6 +745,44 @@ async function getQueryText(chat, initiator) {
     return collapseNewlines(queryText).trim();
 }
 
+/**
+ * Gets common body parameters for vector requests.
+ * @returns {object}
+ */
+function getVectorsRequestBody() {
```

Comment:
Is this ***really*** a get operation, if it persistently **sets** the value inside a global object?

Not like this is a big deal, but maybe a more fitting name here makes sense and is easier to grasp from just reading the function.

Reply:
Ah. Kinda makes sense. It's still less of a `get` and more or a `prepare`, but that's okay.
Just what I first noticed. It's good.